numpy/add_newdocs.py

```diff
@@ -3680,7 +3680,7 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Examples
     --------
-    >>> x = np.random.randint(9, size=(3, 3))
+    >>> x = np.random.randint(low=9, size=(3, 3))
     >>> x
```

Comment:
I'm not sure your comment quite applies here because this isn't even documentation for `np.randint` where you are commenting.